includes/wizards/audience/class-audience-subscriptions.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * Audience Subscriptions Wizard
+ *
+ * @package Newspack
+ */
+
+namespace Newspack;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Audience Subscriptions Wizard.
+ */
+class Audience_Subscriptions extends Wizard {
+	/**
+	 * Admin page slug.
+	 *
+	 * @var string
+	 */
+	protected $slug = 'newspack-audience-subscriptions';
+
+	/**
+	 * Parent slug.
+	 *
+	 * @var string
+	 */
+	protected $parent_slug = 'newspack-audience';
+
+
+	/**
+	 * Get the name for this wizard.
+	 *
+	 * @return string The wizard name.
+	 */
+	public function get_name() {
+		return esc_html__( 'Audience Development / Subscriptions', 'newspack-plugin' );
+	}
+
+	/**
+	 * Add Subscriptions page.
+	 */
+	public function add_page() {
+		add_submenu_page(
+			$this->parent_slug,
+			$this->get_name(),
+			esc_html__( 'Subscriptions', 'newspack-plugin' ),
+			$this->capability,
+			$this->slug,
+			[ $this, 'render_wizard' ]
+		);
+	}
+
+	/**
+	 * Enqueue scripts and styles.
+	 */
+	public function enqueue_scripts_and_styles() {
+		if ( ! $this->is_wizard_page() ) {
+			return;
+		}
+
+		parent::enqueue_scripts_and_styles();
+		wp_enqueue_script( 'newspack-wizards' );
+		wp_localize_script(
+			'newspack-wizards',
+			'newspackAudienceSubscriptions',
+			[
+				'tabs' => [
+					[
+						'path'        => '/configuration',
+						'title'       => esc_html__( 'Configuration', 'newspack-plugin' ),
+						'header'      => esc_html__( 'Manage Subscriptions settings in Woo Memberships', 'newspack-plugin' ),
+						'description' => esc_html__( 'You can manage the details of your subscription offerings in the Woo Memberships plugin.', 'newspack-plugin' ),
+						'href'        => admin_url( 'admin.php?page=wc-settings&tab=memberships' ),
```

Comment:
Is it confirmed that this is the link we want to redirect to? I don't think we change the defaults here. A better destination would be `wp-admin/edit.php?post_type=wc_membership_plan`

Reply:
Refactored in fefbfba19a01815e1522ff62bbb6ceb1cde26289